content/source/docs/enterprise/sentinel/import/tfstate.html.md

```diff
@@ -52,6 +57,24 @@ This returns the list of resources with the given type and name. The
 result is a list because the plan is aware that the count may be greater
 than zero.
 
+### m.data
+
+This returns all the data sources in the module as a map. The map key is the
+type, the value is the same as `m.data.TYPE`.
+
+### m.data.TYPE
+
+This returns all the data sources in the module with the given type as a map
+from name to data source. For the documentation on the value, see
+`m.data.TYPE.NAME`
+
+### m.data.TYPE.NAME
+
+This returns a list of data sources and behaves exactly how a resource lookup
+behaves with the m.resources.TYPE.NAME syntax above. All fields that are
+accessible within resources are available within a data source, save `tainted`,
```

Comment:
I might use "except for" instead of "save", it's less poetic but more common. :)

Reply:
Updated!